examples/rust/components/http-task-manager/README.md

```diff
@@ -0,0 +1,116 @@
+# HTTP Task manager
+
+This web component tracks and updates information about asynchronous tasks, backed by a [Postgres][postgres] Database.
+
+Clients can make HTTP requests to endpoints like the following:
+
+| Endpoint                 | Description                   |
+|--------------------------|-------------------------------|
+| `GET /tasks`             | List ongoing tasks            |
+| `POST /tasks/:id/status` | Get status of a specific task |
+| `POST /tasks`            | Submit a new task             |
+
+Tasks can also be submitted over the wasmCloud Lattice (i.e. [via wRPC][wasmcloud-wrpc]), rather than simply via HTTP.
+
+## Architecture
+
+This application requires a few other pieces to function:
+
+- [wasmCloud HTTP server][provider-http-server] for receiving incoming HTTP requests
+- [wasmCloud SQLDB Postgres Provider][provider-sqldb-postgres] for persistence of tasks
+
+[postgres]: https://www.postgresql.org/
+[wasmcloud-wrpc]: https://wasmcloud.com/docs/reference/glossary#wrpc
+[provider-http-server]: https://github.com/wasmCloud/wasmCloud/tree/main/crates/provider-http-server
+[provider-sqldb-postgres]: https://github.com/wasmCloud/wasmCloud/tree/main/crates/provider-sqldb-postgres
+
+## Prerequisites
+
+- `cargo` >= 1.81
+- [`wash`](https://wasmcloud.com/docs/installation) >=0.30.0
+- `docker` (or some other means to easily run Postgres instances)
+
+## Building
+
+You can build the application with the [WAsmcloud SHell `wash`][wash]:
+
+```console
+wash build
+```
+
+Note that if you'd like to build with `cargo` you need to sepcify the `--target` option:
+
+```console
+cargo build --target=wasm32-wasi
+```
```

Comment:
Thoughts on adding the nightly tag here and targeting wasm32-wasip2?

Reply:
Just thinking mostly that getting a `wasm32-wasi` module is only part of the build picture so may as well depend on nightly & get the real real component

Reply:
In general, I'd like to not require nightly for the build

But ideally would use `wash build`, so they don't have to worry about it -- the reason this is here versus all the other components (which just have `.cargo/config.toml` set up) is that right now the E2E tests need the target to *not* be `wasm32-*` to run properly under `cargo test`